src/Makefile

```diff
@@ -879,6 +879,7 @@ clean_shell_cmd:
 	del *.o /s
 	cd $(RAYLIB_RELEASE_PATH) & \
 	del lib$(RAYLIB_LIB_NAME).a /s & \
+        del lib$(RAYLIB_LIB_NAME).web.a /s & \
```

Comment:
Why? I usually work on Windows with cmd.

Reply:
Ignore my previous messages, I don't know how I look at it that I though it was a line-removal instead of an addition. Sorry for the confusion...